### PR TITLE
GUI Enhancements

### PR DESCRIPTION
**Pull request by dannytayjy**
- Modify GUI to display person list and priority tagging of tasks
- Handle wrap text issues so that name of person, task description and assignees are displayed fully

### CODE REVIEW

File: src/main/java/manageezpz/ui/PersonCard.java [hunk 1]
```diff
@@ -33,17 +33,27 @@
...
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
         email.setText(person.getEmail().value);
+
+        /*List<Task> tasks = model.getFilteredTaskList();
+
+        List<Task> personTasks = tasks.stream()
+                .filter(task -> task.getAssignees().contains(person)).collect(Collectors.toList());
+
+        numOfTasks.setText(personTasks.size() + " Assigned Tasks");*/
```
Comment (Denniszedead) [inline src/main/java/manageezpz/ui/PersonCard.java]: Do remove this comment if you are not using it.

**Review by Denniszedead — DISMISSED** [review]
LGTM! 👍🏻

File: src/main/java/manageezpz/ui/TaskCard.java [hunk 2]
```diff
@@ -78,22 +87,55 @@ public TaskCard(Task task, int displayedIndex) {
...
         }
 
-        if (task.isDone()) {
-            statusIsDone.setText("Done");
-            statusIsDone.getStyleClass().add("cell_status_done_label");
-        } else {
-            statusIsDone.setText("Not Done");
-            statusIsDone.getStyleClass().add("cell_status_not_done_label");
+        // Remove priority row if there are no priority tagged to the task
+        if (task.getPriority().name().equals("NONE")) {
```
Comment (alfredkohhh) [inline src/main/java/manageezpz/ui/TaskCard.java:136]: Thanks for adding this this feature in!

File: src/main/java/manageezpz/ui/TaskCard.java [hunk 3]
```diff
@@ -78,22 +87,55 @@ public TaskCard(Task task, int displayedIndex) {
             employeesTag.setText(assigneesNames);
         }
 
-        // Remove row for 'date/time' if it is a Todo task
+        Image priorityIcon;
+
+        switch (task.getPriority().name()) {
+        case "LOW":
```
Comment (ChanWeiJie) [inline src/main/java/manageezpz/ui/TaskCard.java:93]: Thanks for adding this in !

**Review by ChanWeiJie — APPROVED** [review]
LGTM! 👍

**Pull request merged**